src/views/projects/Patch/PatchTeaser.svelte

```diff
@@ -25,6 +25,12 @@
     latestRevision.base,
     latestRevision.oid,
   );
+
+  // Counts the amount of comments in all the discussions over all revisions.
+  $: commentCount = patch.revisions.reduce(
```

Comment:
Is this actually working, or do we have different kinds of comments? Because I don't see the comment counter in the list view.

http://localhost:3000/seeds/seed.radicle.xyz/rad:z3gqcJUoA1n9HaHKufZs5FCSGazv5/patches/67f8e36cf5f2ecb4635e0b24edf5aefa5dcb4099

<img width="1840" alt="Screenshot 2023-06-27 at 14 29 06" src="https://github.com/radicle-dev/radicle-interface/assets/158411/65f3b54c-0b7e-49d4-82c5-c31cce43272b">

<img width="1840" alt="Screenshot 2023-06-27 at 14 29 10" src="https://github.com/radicle-dev/radicle-interface/assets/158411/7ad34741-01b3-460b-901b-3214381679c6">

Reply:
We should probably rephrase a review without a verdict, maybe `left a review`?

Reply:
Yeah, changing `commented` to `left a review` sounds good.

Reply:
okay did that with commit 2a3e1a8